composer.json

```
@@ -24,7 +25,7 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5",
+        "phpunit/phpunit": "^9.5",
```

```suggestion
        "phpunit/phpunit": "^9.6",
```
All other modules have ^9.6

---

It's not a module and everything is passing... is this necessary? What does 9.6 have that 9.5 doesn't that is needed by this application?